$NetBSD$
Use a more portable invocation of cp(1) which does not follow symlinks.
--- angelscript/projects/gnuc/Makefile.orig	2018-09-23 19:11:04.669903070 -0700
+++ angelscript/projects/gnuc/Makefile	2018-09-23 19:11:18.281487151 -0700
@@ -43,7 +43,7 @@
 INSTALL_SHLIB = $(INSTALL) -m 755
 INSTALL_ARLIB = $(INSTALL) -m 644
 INSTALL_HEADER = $(INSTALL) -m 644
-CP_SYMLINK = cp --no-dereference --preserve=links
+CP_SYMLINK = cp -P
 CP_R = cp -R
 
 
